### PR TITLE
Configure compress level for gzip sink

### LINKED ISSUE #224: Configure compression level on gzip
Right now we can’t. That is lame.

### PR DESCRIPTION
**Pull request by mblcdr**
Resolves #224

### CODE REVIEW

**Comment by swankjesse** [comment]
I was thinking maybe we’d just expose the `Deflater` with an accessor method (`deflater()`) and that’d allow callers to also access stats (`getTotalIn()`, `getTotalOut()`), the dictionary, etc.

Thoughts?

**Comment by mblcdr** [comment]
Sounds like a better idea. Will update the PR accordingly.

File: okio/src/main/java/okio/GzipSink.java [hunk 1]
```diff
@@ -110,6 +110,14 @@ public GzipSink(Sink sink) {
     if (thrown != null) Util.sneakyRethrow(thrown);
   }
 
+  /**
+   * Returns the {@link Deflater}.
+   * Use it to access stats, dictionary, compression level, etc.
+   */
+  public Deflater deflater(){
```
Comment (15characterlimi) [inline okio/src/main/java/okio/GzipSink.java]: missing space before '{'?

**Comment by swankjesse** [comment]
Thanks!

**Pull request merged**